lighthouse-core/test/audits/seo/is-crawlable-test.js

```diff
@@ -0,0 +1,184 @@
+    });
+  });
+
+  it('succeeds when there is no robots metatag', () => {
+    const mainResource = {
+      responseHeaders: [],
+    };
+    const artifacts = {
+      devtoolsLogs: {[IsCrawlableAudit.DEFAULT_PASS]: []},
+      requestMainResource: () => Promise.resolve(mainResource),
+      MetaRobots: null,
+    };
+
+    return IsCrawlableAudit.audit(artifacts).then(auditResult => {
+      assert.equal(auditResult.rawValue, true);
+    });
+  });
+
+  it('fails when page is blocked from indexing with a header', () => {
+    const robotsHeaders = [
+      [
+        {name: 'x-robots-tag', value: 'noindex'},
+      ],
+      [
+        {name: 'X-Robots-Tag', value: 'all'},
+        {name: 'x-robots-tag', value: 'none'},
+      ],
+      [
+        {name: 'X-ROBOTS-TAG', value: 'all, none'},
+      ],
+      [
+        {name: 'x-robots-tag', value: '    noindex    '},
+      ],
+      [
+        {name: 'x-robots-tag', value: 'unavailable_after: 25 Jun 2010 15:00:00 PST'},
+      ],
+      [
+        {name: 'x-robots-tag', value: 'all, unavailable_after: 25-Jun-2010 15:00:00 PST'},
+      ],
+    ];
+
+    const allRuns = robotsHeaders.map(headers => {
+      const mainResource = {
+        responseHeaders: headers,
+      };
+      const artifacts = {
+        devtoolsLogs: {[IsCrawlableAudit.DEFAULT_PASS]: []},
+        requestMainResource: () => Promise.resolve(mainResource),
+        MetaRobots: null,
+      };
+
+      return IsCrawlableAudit.audit(artifacts).then(auditResult => {
+        assert.equal(auditResult.rawValue, false);
+        assert.equal(auditResult.details.items.length, 1);
+      });
+    });
+
+    return Promise.all(allRuns);
+  });
+
+  it('succeeds when there are no blocking directives in the robots header', () => {
+    const mainResource = {
+      responseHeaders: [
+        {name: 'X-Robots-Tag', value: 'all, nofollow'},
+        {name: 'X-Robots-Tag', value: 'unavailable_after: 25 Jun 2045 15:00:00 PST'},
+      ],
+    };
+    const artifacts = {
+      devtoolsLogs: {[IsCrawlableAudit.DEFAULT_PASS]: []},
+      requestMainResource: () => Promise.resolve(mainResource),
+      MetaRobots: null,
+    };
+
+    return IsCrawlableAudit.audit(artifacts).then(auditResult => {
+      assert.equal(auditResult.rawValue, true);
+    });
+  });
+
+  it('succeeds when there is no robots header', () => {
+    const mainResource = {
+      responseHeaders: [],
+    };
+    const artifacts = {
+      devtoolsLogs: {[IsCrawlableAudit.DEFAULT_PASS]: []},
+      requestMainResource: () => Promise.resolve(mainResource),
+      MetaRobots: null,
+    };
+
+    return IsCrawlableAudit.audit(artifacts).then(auditResult => {
+      assert.equal(auditResult.rawValue, true);
+    });
+  });
+
+  it('ignores UA specific directives', () => {
+    const mainResource = {
+      responseHeaders: [
+        {name: 'x-robots-tag', value: 'googlebot: unavailable_after: 25 Jun 2007 15:00:00 PST'},
```

Comment:
ah ok, I was confused about how multiple user agent + a default would be expressed. I didn't realize it'd be duplicate headers rather than a csv

would you mind adding a default value here that's valid just for future readers
i.e.
```
responseHeaders: [
  {name: 'x-robots-tag', value: 'googlebot: unavailable_after: 25 Jun 2007 15:00:00 PST'},
  {name: 'x-robots-tag', value: 'unavailable_after: 25 Jun 2027 15:00:00 PST'},
]
```